### DISCUSSION TITLE
Is your JavaScript code running? Because mine is just creepily staring at me.

### DISCUSSION THREAD

**Discussion by kill3rc** [community/community#145939] [Programming Help]
### Body

```javascript
while(true){
  console.log("What are you staring at?");
}
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by BelakbirSaid** [comment]
![raf,360x360,075,t,fafafa_ca443f4786](https://github.com/user-attachments/assets/2a6af4f9-76c0-4f74-b791-d050fd0fdc4a)

**Comment by mecodeatlas** [comment]
ingHi @DumbestPerson224, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by kill3rc** [reply]
Oh no project in JavaScript I was hoping someone was going to say there code was running, so I could say "Well then you better catch it!"
I will close my question
